crates/weaver_resolver/data/registry-test-4-events/expected-registry.json

```diff
@@ -45,29 +190,354 @@
       }
     },
     {
-      "id": "ios.lifecycle.events",
+      "id": "device.app.lifecycle",
       "type": "event",
-      "brief": "This event represents an occurrence of a lifecycle transition on the iOS platform.\n",
-      "prefix": "ios",
-      "attributes": [
-        0
-      ],
+      "brief": "This event represents an occurrence of a lifecycle transition on Android or iOS platform.\n",
+      "note": "This event identifies the fields that are common to all lifecycle events for android and iOS using the `android.state` and `ios.state` fields. The `android.state` and `ios.state` attributes are mutually exclusive.\n",
+      "stability": "experimental",
+      "attributes": [],
       "name": "device.app.lifecycle",
       "lineage": {
         "source_file": "data/registry-test-4-events/registry/mobile-events.yaml"
+      },
+      "body": {
+        "name": "device.app.lifecycle.fields",
+        "type": "map",
+        "requirement_level": "required",
+        "fields": [
+          {
+            "name": "ios.state",
+            "type": "enum {active, inactive, background, foreground, terminate}",
```

Comment:
Also for AnyValue comment https://github.com/open-telemetry/weaver/pull/365#discussion_r1767768808

This becomes (`id` instead of `name` and the `enum` for the type instead of `enum {active, inactive, background, foreground, terminate}`

```json
      "body": {
        "id": "device.app.lifecycle.fields",
        "type": "map",
        "requirement_level": "required",
        "fields": [
          {
            "id": "ios.state",
            "type": "enum",
            "members": [
              {
                "id": "active",
```